src/Core/Utilities/StrictEmailAddressAttribute.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Text.RegularExpressions;
+
+namespace Bit.Core.Utilities
+{
+    public class StrictEmailAddressAttribute : ValidationAttribute
```

Comment:
Why not inherit from `EmailAddressAttribute`?

Reply:
I tried to do this initially, but the `EmailAddressAttribute` is `sealed`, so you can't inherit from it:
`public sealed class EmailAddressAttribute
  : DataTypeAttribute`

Reply:
BAH! Sorry, I wasn't sure but drat, not sure _why_ they choose to mark some of these classes as _sealed_ but oh well, yeah then you're good, thanks Thomas! At least you had the right idea in the beginning; thanks!